kyverno.tf

```diff
@@ -0,0 +1,107 @@
+locals {
+  kyverno = merge(
+    local.helm_defaults,
+    {
+      name                   = "kyverno"
+      namespace              = "kyverno"
+      chart                  = "kyverno"
+      repository             = "https://kyverno.github.io/kyverno/"
+      create_ns              = false
+      enabled                = false
+      chart_version          = "1.3.3"
+      default_network_policy = true
+    },
+    var.kyverno
+  )
+
+  values_kyverno = <<VALUES
+podSecurityStandard: restricted
+validationFailureAction: audit
```

Comment:
Also this is already the default so we can remove it

Reply:
I thought about it when writing it up, making the `audit` mode explicit on default aimed to prevent any unexpected change if Kyverno changes the default to `enforce` (I highly doubt this could happen, but it would be quite a bad surprise)

We can make this setting implicit and rely on Kyverno if you prefer to delegate to their recommended defaults

Reply:
I don't have a strong opinion on this, we can leave it hardcoded but it add "unnecessary" code. Leaving it can also prevent change on the chart side so it can be a good thing but I'd like to thing that they are not going to change from `audit`  to `enforce` without majoring the chart version and risk breaking all of their user cluster